pubspec.yaml

```diff
@@ -4,8 +4,8 @@
 
 name: ffigen
 version: 5.0.0-dev.1
-homepage: https://github.com/dart-lang/ffigen
```

Comment:
You can't leave this? The docs say that homepage can also be a repository:
https://dart.dev/tools/pub/pubspec

Reply:
Yeah, but it doesn't add a lot of value to have both if they both point to the same thing. We want to have the `repository` field populated for core dart packages - it'll let us additional automation and tooling here. If this package had a website that was distinct from the repo url, then I'd say keep both. But given that they're the same, users would just see the same link twice in the pub page:

<img width="328" alt="Screen Shot 2022-04-18 at 6 44 36 PM" src="https://user-images.githubusercontent.com/1269969/163903600-fce93ec6-1589-41b2-990e-02d4598e3ef0.png">

Reply:
That makes sense. 👍